administrator/components/com_cookiemanager/cookiemanager.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<extension type="component" method="upgrade">
+	<name>com_cookiemanager</name>
+	<author>Joomla! Project</author>
+	<creationDate>June 2021</creationDate>
+	<copyright>(C) 2021 Open Source Matters, Inc.</copyright>
```

Comment:
The Copyright notice should be:
`<copyright>(C) 2005 - 2021 Open Source Matters. All rights reserved.</copyright>`

Reply:
Nope, that's not right The year is the year of first appearance in the public. We made a great effort to change that format to match the standard. So no need to change here.

Reply:
Thanks @nibra